proto/kvrpcpb.proto

```diff
@@ -176,12 +176,12 @@ message PrewriteRequest {
 }
 
 // Used to update the lock_ttl of a large transaction to prevent it from been killed.
-message LargeTxnHeartBeatRequest {
+message TxnHeartBeatRequest {
     bytes primary_lock = 1;
```

Comment:
Need `Context context = 1;` like other requests.

Reply:
Does it need a `lock_ts`?